source/sessions/tests/README.md

```diff
@@ -29,6 +29,15 @@ As part of the test setup for these cases, create a `MongoClient` pointed at the
 in the test case and verify that the test server does NOT define a value for `logicalSessionTimeoutMinutes` by sending a
 hello command and checking the response.
 
+## Specific operations and behaviour for unified tests
+
+An operation on sessions called `getSnapshotTime` must be supported for unified tests. This operation returns the value
```

Comment:
Sorry for returning with this again, but I've just realized that we should mirror this changes into unified test format spec:
here: https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.md#session-operations
and here: https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.md#entity_session

Reply:
I've mirrored this in the Session Operations section in unified test format spec. I did not modify the Entity Session section, as it already refers to the single spec documents, where I've already added the necessary info.